frontends/p4/validateParsedProgram.cpp

```diff
@@ -180,6 +180,13 @@ void ValidateParsedProgram::postorder(const IR::SwitchStatement* statement) {
     auto inAction = findContext<IR::P4Action>();
     if (inAction != nullptr)
         ::error("%1%: switch statements not allowed in actions", statement);
+    bool defaultFound = false;
+    for (auto c : statement->cases) {
+        if (defaultFound)
```

Comment:
Can we please ensure that we print the error only once per select statement? And we also print the offending labels.

Reply:
We are printing the offending label. Do you also want to see the default label?

Reply:
The test case stderr shows the output.

Reply:
My bad on the label. I looked at the travis run and it shows:
```error: SwitchCase: label following default label
error: SwitchCase: label following default label
error: SwitchCase: label following default label```

I interpreted SwitchCase as the message.
Also, there are no line numbers printed.

Reply:
Yes, the line number is the one in parentheses.
The error function does this automatically.
```
default-switch.p4(30): error: SwitchCase: label following default label
            b: { return; }
            ^^^^^^^^^^^^^^
default-switch.p4(31): error: SwitchCase: label following default label
            c:
            ^
```

Reply:
I see the following output on Travis:
```
978/1166 Test #117: bmv2/testdata/p4_14_samples/04-GatewayDefault.p4 ..............................***Failed    0.46 sec
Error compiling
error: SwitchCase: label following default label
error: SwitchCase: label following default label
*** Test failed
```